### ISSUE TITLE
Breadcrumb links are not accessible via keyboard (tab)

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2263]
To reproduce the issue go to: https://modus-web-components.trimble.com/?path=/docs/components-breadcrumb--default#default

Press <kbd>Tab</kbd> and I'd expect the 'Crumb 1' link to be focused, but because it doesn't have the `href` attribute then it is skipped.
